netlify/functions/manage-findings.ts

```diff
@@ -38,37 +38,29 @@ import { apiKey, domain } from "../_config";
 
 async function getFinding(
   username: string,
-  contest: Contest,
+  repoName: string,
   issueId: number
 ): Promise<Response> {
   const client = new Octokit({ auth: process.env.GITHUB_TOKEN });
 
-  const submission_files = (
-    await getAvailableFindings(client, username, contest)
+  const allFindings = await getAllFindings(
+    client,
+    repoName,
+    process.env.GITHUB_CONTEST_REPO_OWNER!
+  );
+
+  // todo: don't rely on full listing
```

Comment:
isn't this `todo` now done?

Reply:
No, this is actually an old comment from when the logic was originally written. The idea here is that we could just fetch the single issue and data file from GH, but in fact we get all findings, then filter by warden, and then filter by the issue number we want. 

It's certainly inefficient, but this function should only be called if the user navigates directly to `contests/[contest-slug]/submit?issue=[issueNumber]` (which is not common). If they navigate by clicking "edit" from the findings tab on the contest page, the finding data is just passed through to the form via location state.